xwiki-platform-core/xwiki-platform-tree/xwiki-platform-tree-war/src/main/webapp/templates/tree_macros.vm

```diff
@@ -37,6 +37,9 @@
 
 #macro (tree $options)
   <div #treeAttributes($options)></div>
+  <span id="xtree#if($!{macro.options.id})-$!{macro.options.id}#{end}-description" class="sr-only">
```

Comment:
I don't think we should force an ID prefix. The default ID can have a prefix (see my other comment), but if the user specifies an ID, we should use it as is, i.e. we should only add the ``-description`` suffix.